code-of-conduct.md

```diff
@@ -0,0 +1,88 @@
+# Dask Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, sex characteristics, gender identity and expression,
+level of experience, education, socio-economic status, nationality, personal
+appearance, race, religion, or sexual identity and orientation. The following list is
```

Comment:
Shouldn't it be "the preceding list" rather than "the following list"?

Reply:
Or perhaps the list above should be terminated thusly: """race, religion, sexual identity and orientation, or any other personal characteristic""".

Reply:
> Or perhaps the list above should be terminated thusly: """race, religion, sexual identity and orientation, or any other personal characteristic"""

I'm very happy with this proposal, thank you for the suggestion @pitrou. There have been several comments related to the phrasing of this particular sentence and I think ending with "or any other personal characteristic" is a nice way of addressing the concerns that have been raised. @jakirkham @guillaumeeb @pentschev you've all commented on this recently, are there any objections to making the proposed change?